draft-sharabayko-mops-srt.md

```diff
@@ -1100,6 +1103,45 @@ Control Information Field (CIF).
 : A single value or a range of lost packets sequence numbers. See packet sequence number
 coding in {{packet-seq-list-coding}}.
 
+### Congestion Warning {#ctrl-pkt-congestion}
+
+The Congestion Warning control packet is not in use by SRT sender
+at the moment and reserved for the future.
+However, the receiver handles this type of packet if it ever arrives.
+
+If the receiver receives this packet, it slows down sending rate
+increasing the minimum inter packet sending interval by 12.5%.
```

Comment:
This description (1108 - 1113) sounds to me like it is implementation-specific. Maybe we can make it less so:

```suggestion
The Congestion Warning control packet is reserved for future use. Its purpose is to signal a receiver to increase the minimum inter-packet sending interval by a discrete value in order to slow down the sending rate.
```

Also, isn't it the sender that sets the minimum inter-packet sending interval?

Reply:
> Also, isn't it the sender that sets the minimum inter-packet sending interval?

Right!!! I mixed up "control packet receiver" and "receiver".
Should be: "If a sender receives this Congestion Warning..."